### ISSUE TITLE
[MySQL]: Error parsing `INSERT IGNORE INTO` query

### ISSUE THREAD

**Issue by tbantle22** [taozhi8833998/node-sql-parser#1089]
Describe the bug
Should parse `INSERT IGNORE INTO tablename VALUES (1, 'hi')` query.

An `INSERT IGNORE INTO` query that specifies columns parses correctly (i.e. `INSERT IGNORE INTO tablename (pk, col1) VALUES (1, 'hi')`). The same query without the `IGNORE` also parses correctly (i.e. `INSERT INTO tablename VALUES (1, 'hi')`).

Database Engine
which database engine does your SQL running on? such as MySQL, PostgreSQL.
MySQL

To Reproduce
-the SQL that be parsed
-the node-sql-parser version ^4.3.0"
-the node version

```
query: insert ignore into `tablename` values (1, 'hi')

error: SyntaxError: Expected "#", "(", "--", ".", "/*", "PARTITION", or [ \t\n\r] but "v" found.
```

**Comment by taozhi8833998** [comment]
@tbantle22 I have fixed it and will publish a new version this week. Thank you for pointing it out.

**Comment by taozhi8833998** [comment]
@tbantle22 fixed in the `4.5.0` version. Thanks for waiting so long.
